vulcanai2/models/cnn.py

```diff
@@ -99,17 +81,17 @@ def _create_network(self, **kwargs):
             self._create_classification_layer(
                 self.conv_flat_dim, kwargs['pred_activation'])
             
-            if torch.cuda.is_available():
-                for module in self.modules():
-                    module.cuda()
+        if torch.cuda.is_available():
+            for module in self.modules():
+                module.cuda()
 
     def _create_classification_layer(self, dim, pred_activation):
         self.network_tail = DenseUnit(
             dim, self.out_dim, activation=pred_activation)
 
-    def forward(self, x):
+    def _forward(self, x, **kwargs):
```

Comment:
same comment as above about turning forward into a hidden function. isn't `forward` required for any nn.Module object?